tests/models/bridgetower/test_modeling_bridgetower.py

```diff
@@ -54,87 +60,169 @@
     from transformers import BridgeTowerProcessor
 
 
-class BridgeTowerModelTester:
+class BridgeTowerTextModelTester:
```

Comment:
There is no `BridgeTowerTextModelTest` however: we just use this tester class to create text config and text inputs